grayskull/utils.py

```diff
@@ -64,3 +65,12 @@ def get_local_modules(sdist_folder: str) -> List:
             continue
         result.append(os.path.splitext(py_file)[0])
     return result
+
+
+def ensure_list(arg) -> List:
+    if isinstance(arg, types.StringTypes) or not hasattr(arg, "__iter__"):
+        if arg is not None:
+            arg = [arg]
+        else:
+            arg = []
+    return arg
```

Comment:
I think we can try to simplify 
```suggestion
    """Convert object to a list
    
    :param arg: Python object which will be converted to a list
    :return: Object converted
    """
    if arg is None:
        return []
    if not isinstance(arg, list):
        return [arg]
    return arg
```
What do you think?

and could you please just add a unit test to cover this help function would be nice :)